app/src/test/java/org/mozilla/fenix/share/ShareInteractorTest.kt

```diff
@@ -68,4 +68,11 @@ class ShareInteractorTest {
 
         verify { controller.handleShareToApp(app) }
     }
+
+    @Test
+    fun onSaveToPDF() {
+        interactor.onSaveToPDF("tabID")
+
+        verify { controller.handleSaveToPDF("tabID") }
+    }
```

Comment:
:car: nit: I think we are trying to write all new tests in the `GIVEN WHEN THEN` style even if they do not match the rest of a file